### DISCUSSION TITLE
how to open a ipynb(jupyter notebook) file in code-server

### DISCUSSION THREAD

**Discussion by RizzoYN** [coder/code-server#3446] [Q&A] [answered]
i want to open a jupyter notebook file in code-server today, but it can not be loaded anyway ,so if i want to edit *.ipynb in code-server, how should i do ?

**Comment by code-asher** [comment, marked as answer]
There are currently a few issues with the Python and Jupyter extensions in our marketplace so I would try the following:

1. Remove the Python and Jupyter extensions if already installed
2. Install Python manually from https://open-vsx.org/extension/ms-python/python
3. Refresh the page

From there you should be able to open .ipynb files in code-server.
